packages/server/test/helpers.ts

```diff
@@ -124,7 +124,8 @@ after(async function() {
 });
 
 beforeEach(() => {
-  actions.atom.reset(initial => ({ ...initial, manifest, bundler }));
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  (actions.atom as any)._reset((initial: OrchestratorState) => ({ ...initial, manifest, bundler }));
```

Comment:
Can we perhaps extract this into a test helper that comes with the atom package? Like `resetAtom()` and then invoke it from here?

Reply:
@cowboyd I like that idea a lot.  On its way